zkstack_cli/crates/config/src/forge_interface/gateway_ecosystem_upgrade/input.rs

```diff
@@ -33,6 +34,8 @@ impl GatewayEcosystemUpgradeInput {
             era_chain_id,
             testnet_verifier,
             owner_address: current_contracts_config.l1.governance_addr,
+            // FIXME - where should it be loaded from?
+            governance_upgrade_timer_initial_delay: 15,
```

Comment:
in local testing providing even 0 is fine

Reply:
In prod I would expect us manually compiling the input files for foundry scripts